lib/ansible/plugins/action/net_config.py

```diff
@@ -36,8 +36,8 @@ class ActionModule(ActionBase):
 
     TRANSFERS_FILES = False
 
-    def run(self, tmp=None, task_vars=None):
-        result = super(ActionModule, self).run(tmp, task_vars)
+    def run(self, tmp=None, task_vars=None, path=None):
```

Comment:
This isn't where you are going to get the path information.  Most likely the backup path will be passed from the module in which case you will find that information in `_task_args`.